File: sample/OpenCvSharpSamplesVB/Samples/CaptureCamera.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports OpenCvSharp

' Namespace OpenCvSharpSamplesVB
    ''' <summary>
    ''' カメラのキャプチャ
    ''' </summary>
    Friend Module CaptureCamera
        Public Sub Start()
            Using cap As CvCapture = CvCapture.FromCamera(0) ' device type + camera index
                Using w As New CvWindow("SampleCapture")
                    Do While CvWindow.WaitKey(10) < 0
                        w.Image = cap.QueryFrame()
                    Loop
                End Using
            End Using
        End Sub
    End Module
' End Namespace

File: sample/CStyleSamplesCS/Samples/CaptureCamera.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using OpenCvSharp;

namespace CStyleSamplesCS
{
    /// <summary>
    /// カメラのキャプチャ
    /// </summary>
    class CaptureCamera
    {
        public CaptureCamera()
        {
            using (CvCapture cap = CvCapture.FromCamera(0)) // device type + camera index
            using (CvWindow w = new CvWindow("SampleCapture"))
            {
                while (CvWindow.WaitKey(10) < 0)
                {
                    w.Image = cap.QueryFrame();
                }
            }
        }
    }
}